todo_app/app.py

```diff
@@ -7,16 +7,26 @@
 import os
 from flask_login import LoginManager, login_required, current_user
 from todo_app.security.Auth import Auth
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
+
 
 
 def create_app(db_name=None):
     app = Flask(__name__)
     app.config.from_object(Config())
+    app.logger.setLevel(app.config['LOG_LEVEL'])
+    if app.config['LOGGLY_TOKEN'] is not None:
+        handler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{app.config["LOGGLY_TOKEN"]}/tag/todo-app')
+        handler.setFormatter(Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s"))
+        app.logger.addHandler(handler)
     MongoDbApi.init(db_name)
     Auth.init()
 
     def can_write():
-        return app.config['LOGIN_DISABLED'] or current_user.has_write_permission()
+        can_current_user_write = app.config['LOGIN_DISABLED'] or current_user.has_write_permission()
+        app.logger.info("Does current user have write permission? %s", can_current_user_write)
```

Comment:
Is this a useful message?

I suspect that it would benefit a lot from at least including the user id if it is something we want to log, as otherwise when our app has more than one user we'll be struggling to learn anything of interest from this log!